posthog/api/notebook.py

```diff
@@ -241,17 +241,19 @@ def _filter_request(self, request: request.Request, queryset: QuerySet) -> Query
                 # each of those objects can have attrs which is a dict with id for the recording
                 for match_pair in match_pairs:
                     target, match = match_pair.split(":")
-                    if target == "recording":
+                    if target:
                         if match == "true":
                             queryset = queryset.filter(content__content__contains=[{"type": f"ph-{target}"}])
                         elif match == "false":
                             queryset = queryset.exclude(content__content__contains=[{"type": f"ph-{target}"}])
                         else:
+                            queryset = queryset.filter(content__content__contains=[{"type": f"ph-{target}"}])
+
+                            id = "id"
+                            if target == "replay-timestamp":
+                                id = "sessionRecordingId"
```

Comment:
I wonder if we should consider having `id` for that component as well. @daibhin and I had a back and forth on this and one of my arguments in favour of a standard `id` field mean things like this could be simpler.